transition-scripts/deploy-gold-in-standby.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+usage() {
+    cat <<EOF
+Deploy Keycloak resources in the target Gold namespaces in standby mode.
+This is for use in the case that gold was not able to be put into standby mode.
+The user will have to scale Patroni-Gold pods to zero before running. This script will
+delete existing patroni PVCs and configmaps.
+
+Usages:
+    $0 <namespace>
+
+Available namespaces:
+    - c6af30-dev
+    - eb75ad-dev
+    - eb75ad-test
+    - eb75ad-prod
+
+Pre-conditions:
+    - the patroni cluster in Golddr must be healthy and in active mode.
+    - the Gold patroni pods are scaled to zero.
+
+Examples:
+    $ $0 c6af30-dev
+EOF
+}
+
+if [ "$#" -lt 1 ]; then
+    usage
+    exit 1
+fi
+
+namespace=$1
+
+pwd="$(dirname "$0")"
+source "$pwd/helpers/_all.sh"
+
```

Comment:
let's make sure `golddr` is running as active and health in code even though we put that as a pre-condition up there.

Reply:
good idea, that makes things more robust